.travis.yml

```diff
@@ -0,0 +1,4 @@
+language: ruby
+rvm:
+  - 2.2.3
```

Comment:
Have you consider add latest versions like '2.2' and '2.3'? Also to add 'ruby-head' to know that is compatible with the latest ruby?